blueprints/app/files/tests/helpers/module-for-acceptance.js

```diff
@@ -11,12 +11,12 @@ export default function(name, options = {}) {
       this.application = startApp();
 
       if (options.beforeEach) {
-        return options.beforeEach.apply(this, arguments);
+        return options.beforeEach.apply(...arguments);
```

Comment:
See https://github.com/DockYard/ember-suave/issues/109 for some additional details.